tools/render_junit.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import typing
+
+try:
+    import junitparser
+except ImportError:
+    raise ImportError(
+        "junitparser not found, please install with 'pip install junitparser'"
+    )
+
+try:
+    import rich
+except ImportError:
+    print("rich not found, for color output use 'pip install rich'")
+
+def parse_junit_reports(path_to_reports: str) -> typing.List[junitparser.TestCase]:
```

Comment:
Typical pattern is to import just a specific elements from `typing`
```suggestion
def parse_junit_reports(path_to_reports: str) -> List[junitparser.TestCase]:
```